DecipherNow/gm-fabric-dashboard
# Refactor AppHeader/scenes

Resolves #1282 

## Proposed Changes

  - Moved InstanceHeaderContent from components to Appheader/scenes
  - Removed UseSDS and NoSDS folders and replaced with HeaderContent scenes
  
## How to Test

  - View folder structure in editor
  - Click around app to double check that there are no breaking changes
